org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -105,17 +120,16 @@ ${signature.Name} = Signature("${signature.Name}", "${signature.Multiplicity}",
 # --- Conc States ---
 #macro(generateState $concState $padding)
 ${padding}class ${concState.getName()}_State(State):
-$padding     # substates
+
+$padding    # substates
 	#foreach($substate in $concState.getSubstates())
 		#set($tab = "    ")
 		#set($newpadding = "$padding$tab")
 		#generateState($substate $newpadding)
 	#end
 
-$padding    """ The wrapper state for ${concState.getName()} """
-
 $padding    def __init__(self, context: State, root_context: State) -> None:
-$padding        super(${concState.getName()}, self).__init__(self) # init superclass
+$padding        super({concState.getName()}_State, self).__init__(self) # init superclass
```

Comment:
Missed `$` in front of `{concState.getName()}`

Reply:
Good catch. Fixed.